development/cpp/custom_modules_in_cpp.rst

```diff
@@ -387,8 +387,8 @@ during runtime with the ``LD_LIBRARY_PATH`` environ variable:
 
 .. code-block:: shell
 
-    user@host:~/godot$ export LD_LIBRARY_PATH=`pwd`/bin/
-    user@host:~/godot$ ./bin/godot*
+    export LD_LIBRARY_PATH=`pwd`/bin/
+    ./bin/godot*
```

Comment:
FWIW I tend to use the following non-`export` form which I like because it's a one-liner & doesn't "pollute the environment":

```
LD_LIBRARY_PATH=`pwd`/bin/ ./bin/godot*
```

It also has the benefit that the required environment variable configuration is kept with the command.

Reply:
There's a warning just below that recommends using `export` so that the variable is set when running the project from the editor.

That said, I'll change the syntax to make the variable work if the path contains spaces (right now, it won't).